service/worker/scheduler/workflow_test.go

```diff
@@ -1669,6 +1670,69 @@ func (s *workflowSuite) TestUpdateNotRetroactive() {
 	)
 }
 
+func (s *workflowSuite) TestUpdateWithJitter() {
```

Comment:
I do not understand what is the assertion in this test.

Reply:
See the "Why?" in the PR description:

> Fixes #4866. Fixes the situation where an update happens in between the nominal time and the jittered time of an action.

and then see line 1714

Reply:
May be we can call it TestWhenUpdateBetweenNominalAndJitteredTimeWillXXX?